### ISSUE TITLE
add installation guide to readme

### ISSUE THREAD

**Issue by quanvo87** [IBM-Swift/Swift-SMTP#84]

**Comment by parker1010** [comment]
Please add an installation guide asap! I'd like to use this module in my xCode project but don't know how!

**Comment by quanvo87** [comment]
@parker1010 installation instructions have just been added! Please check out the README and let me know if you have any issues.

**Comment by parker1010** [comment]
@quanvo87 Thanks for the quick response! I generated a new Xcode project with the command "swift package generate-xcodeproj" but that new project won't run the Simulator. After researching about Swift Package Manager I found: "The generated project is ideal for unit tests. It does not include the overhead of launching the iOS simulator or a target application." Any advice for people like me who want to use your module and still run the Simulator in their project?

**Comment by parker1010** [comment]
@quanvo87 Yes, I'm learning iOS app development. Installation instructions using CocoaPods would be very helpful for me as I'm already using CocoaPods (for Google Firebase) in my current iOS project.

**Comment by quanvo87** [comment]
I'll work on adding CocoaPods support sometime this week.

**Comment by quanvo87** [comment]
Unfortunately adding dual support for SPM and CocoaPods is proving to be difficult for me to figure out. This feature may not added in the near future as I can't commit too much time to this right now. Any pointers would be appreciated!

**Comment by deepakyadeedya** [comment]
Hi, I need to add a .txt file in the background I mean without the user interface...! If you need I will share my code what I'm doing please help.
